nuitka/tools/testing/OutputComparison.py

```diff
@@ -82,7 +82,9 @@ def normalizeTimeDiff(outputStr):
     =================== 1059 passed, 8 warnings in x.xx seconds ===================
     """
 
-    matchObj = re.search(b"in [0-9]+.[0-9][0-9] seconds", outputStr)
+    matchObj = re.search(b"in [0-9]+.[0-9][0-9]s", outputStr) or re.search(
```

Comment:
Are you familiar wird regular expressions? Because that kind of `or` is now how you use them, instead you make am or inside the re, let me know if you need help with that.

Reply:
`matchObj = re.search(b"in [0-9]+.[0-9][0-9](s| seconds)", outputStr)`
Something like that?

Reply:
Looks good to me, should work.